### PR TITLE
android.R.id.home also returns OnBackPressed

### PR DESCRIPTION
**Pull request by borjaruiz**
Now "shouldReturnOkOnBackPressed" works when user clicks toolbar arrowback too.

### CODE REVIEW

File: leku/src/main/java/com/schibstedspain/leku/LocationPickerActivity.java [hunk 1]
```diff
@@ -258,7 +258,8 @@ public boolean onCreateOptionsMenu(Menu menu) {
   public boolean onOptionsItemSelected(MenuItem item) {
     int id = item.getItemId();
     if (id == android.R.id.home) {
-      returnCurrentPosition();
+      //returnCurrentPosition();
```
Comment (ferranpons) [inline leku/src/main/java/com/schibstedspain/leku/LocationPickerActivity.java]: It's OK but please remove the comment. Thanks!

**Comment by borjaruiz** [comment]
Perfect!

**Comment by ferranpons** [comment]
@CristianGM we have a stopper here. Can we bypass the tests for now until we have trusted UI tests?

**Comment by ferranpons** [comment]
@borjaruiz new version 2.3.0 is available for download. Enjoy!

**Comment by borjaruiz** [comment]
thx!

2016-10-25 12:17 GMT+02:00 Ferran Pons <notifications@github.com>:

**Pull request merged**